packages/editor-ui/src/plugins/codemirror/completions/nativesAutocompleteDocs/luxon.instance.docs.ts

```diff
@@ -357,47 +494,121 @@ export const luxonInstanceDocs: Required<NativeDoc> = {
 				section: 'other',
 				hidden: true,
 				docURL: 'https://moment.github.io/luxon/api-docs/index.html#datetimeresolvedlocaleoptions',
-				returnType: 'object',
-				args: [{ name: 'opts', type: 'object' }],
+				returnType: 'Object',
+				args: [{ name: 'opts', type: 'Object' }],
 			},
 		},
 		set: {
 			doc: {
 				name: 'set',
+				description: i18n.baseText('codeNodeEditor.completer.luxon.instanceMethods.set'),
 				section: 'edit',
 				docURL: 'https://moment.github.io/luxon/api-docs/index.html#datetimeset',
 				returnType: 'DateTime',
-				args: [{ name: 'values', type: 'object' }],
+				args: [
+					{
+						name: 'values',
+						optional: false,
+						description: i18n.baseText(
+							'codeNodeEditor.completer.luxon.instanceMethods.set.args.values',
+						),
+						type: 'Object',
+					},
+				],
+				examples: [
+					{
+						example: "dt = '2024-03-30T18:49'.toDateTime()\ndt.set({ year:1982, month:10 })",
+						evaluated: '1982-10-20T18:49',
```

Comment:
It does with everything set to `0`, but I wonder how to best convey that this is a `DateTime` object and not just a string?

`[DateTime:1982-10-20T18:49]`, or same as in the real expression output `[Object: "2024-05-15T04:55:04.318-04:00"]`, or something else?

Reply:
Likely a Giulio question but personally I'd go for a middle ground `[DateTime: "2024-05-15T04:55:04.318-04:00"]` because with datetimes I'd like an exact preview of all the components and what this will look like stringified, e.g. if the user intends to use the timezone and we exclude it to trim down verbosity, we're probably not doing them any favors?